lib/StaticUniformGrid.js

```diff
@@ -0,0 +1,477 @@
+'use strict';
+var Cesium = require('cesium');
+var AxisAlignedBoundingBox = Cesium.AxisAlignedBoundingBox;
+var Cartesian3 = Cesium.Cartesian3;
+var defineProperties = Cesium.defineProperties;
+var clone = require('clone');
+var triangleAxisAlignedBoundingBoxOverlap = require('./triangleAxisAlignedBoundingBoxOverlap');
+
+module.exports = StaticUniformGrid;
+
+/**
+ * A Uniform Grid for unmoving objects.
+ * @alias StaticUniformGrid
+ * @constructor
+ *
+ * @param {Object[]} objects Objects that the grid should store.
+ * @param {Number} cellWidth, The cell width of the uniform grid.
+ * @param {StaticUniformGrid~CompareObjectBoundingBox} compareObjectBoundingBox
+ * @param {StaticUniformGrid~CellCheckFunction} cellCheck
+ */
+
+var minScratch = new Cartesian3();
+var maxScratch = new Cartesian3();
+var halfDimensionsScratch = new Cartesian3();
+
+function StaticUniformGrid(data, cellWidth, compareObjectBoundingBox, cellCheck) {
```

Comment:
Throughout this file `data`, `objects`, and `items` are used interchangeably. I would pick one name to use.